vspace/src/main/webapp/WEB-INF/views/staff/spaces/spacelist.jsp

```diff
@@ -3,6 +3,79 @@
 <%@ taglib prefix="sec"
 	uri="http://www.springframework.org/security/tags"%>
 
+<script>
+//# sourceURL=click.js
+
+function checkSpaceLinkPresent(spaceId){  
+    
+    var urlToLoadOnSuccess = "<c:url value="/staff/space/list"/>";
+    var urlToLoadOnError = "<c:url value="/staff/space/list"/>";
+    var dataUrl = "<c:url value="/staff/space/" />" + spaceId + '?${_csrf.parameterName}=${_csrf.token}';
+    var warningMessage = "Warning! Other spaces have links to this space! Do you still want to delete?";  
```

Comment:
same comments as for other jsp file apply here; actually let's put it into it's own jsp file and include it since it seems to be the same code